test/csuite/import/smoke.sh

```diff
@@ -0,0 +1,56 @@
+#! /bin/sh
+
+set -e
+
+# Bypass this test for slow machines, valgrind
+test "$TESTUTIL_SLOW_MACHINE" = "1" && exit 0
+test "$TESTUTIL_BYPASS_VALGRIND" = "1" && exit 0
+
+# If $top_builddir/$top_srcdir aren't set, default to running in test/csuite.
```

Comment:
I guess the idea is to assume the default location of calling smoke.sh is from within `build_posix/test/csuite`. The `test/csuite` in comment makes me think of a different directory outside of `build_posix`.

Reply:
`build_posix` is not required. I personally never build in `build_posix`, I always build in the top-level directory.

Reply:
That's a good point. Though in evergreen.yml it's set to build under `build_posix`, other directories  are supported as well. Ignore my comment then.